apps/authentication/api/mfa.py

```diff
@@ -23,8 +23,6 @@
 ]
 
 
-
-
 # MFASelectAPi 原来的名字
 class MFASendCodeApi(AuthMixin, CreateAPIView):
     """
```

Comment:
I'm sorry, but you haven't provided me with any specific code that I can analyze. Could you please provide the code or some snippets related to the topic?